examples/satellite-azure/inputs.tfvars.template

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+## IBM Cloud Variables
+##############################################################################
+# Set the variable export TF_VAR_ibmcloud_api_key=
+#ibmcloud_api_key = ""
+ibm_resource_group = "default"
+
+##############################################################################
+## Azure Variables
+##############################################################################
+client_id       = ""
+tenant_id       = ""
+client_secret   = ""
+subscription_id = ""
+
+# Standard_D16ds_v4: 16 vCPU - 64 GB AM
+instance_type = "Standard_D16ds_v4"
```

Comment:
since this is a template, it doesn't really matter, but `instance_type` is deprecated instead use `cp_hosts` similar to `addl_hosts` which allows you to enter the amount of his instance_type you want.

Reply:
I removed it.